tests/stakeholder/test_vis_stakeholder.py

```diff
@@ -11,10 +11,11 @@
 import os
 import importlib.resources
 
-relative_tolerance = 10 ** (-12)
+# relative_tolerance = 10 ** (-12)
+relative_tolerance = 10 ** (-6)
```

Comment:
I reduced the relative tolerance because the tests started failing when I added the other partitioning scheme to the tests. This is to be expected because the order of adding the visibilities will be different between the partitioning schemes.